ec2net-functions

```diff
@@ -176,6 +208,11 @@ rewrite_primary() {
   fi
   logger --tag ec2net "[rewrite_primary] Rewriting configs for ${INTERFACE}"
   cidr=$(get_cidr)
+  if [ "${?}" -gt 0 ]; then
+    # For any errors from IMDS, bail out early rather than rewriting anything
+    # We'll get back here later and be able to rewrite things.
+    return $?
```

Comment:
I know the error gets logged in get_meta...60 times. Should we add another log entry here as well, now that we know we are bailing out?

Reply:
I'm feeling guilty about logging it in `get_meta` now.... But you're probably right here.

I'd be inclined to drop the 60x "retrying" log messages and just have one when bailing out.